api/pagination.go

```diff
@@ -9,5 +9,7 @@ type PaginationResponse struct {
 	Page  int `json:"page,omitempty"`
 	Limit int `json:"limit,omitempty"`
 
-	//TODO: add some indication of number of records/pages or if a next page exists
+	TotalCount int  `json:"total_count,omitempty"`
+	TotalPages int  `json:"total_pages,omitempty"`
+	HasNext    bool `json:"has_next,omitempty"`
```

Comment:
would suggest `hasNext` based on our [API guidelines](https://github.com/infrahq/infra/blob/main/docs/api/guidelines.md#field-naming)

Reply:
Should we also rename `pagination_info`? It is technically breaking, but very likely unused since pagination is currently optional and buried in the API.

Reply:
Great catch. Since we haven't published paginationInfo as part of our API docs or referenced we are okay to edit this to `paginationInfo` for consistency